src/SCardCall.hpp

```diff
@@ -48,45 +48,39 @@ void SCardCall(const char* callerFunctionName, const char* file, int line,
 {
     // TODO: Add logging - or is exception error message enough?
 
-#ifdef __APPLE__
-    // Apple version of pcsc-lite uses unsigned code constants
-    const uint32_t result = scardFunction(args...);
-#else
-    // Standard version of pcsc-lite uses signed code constants
     const LONG result = scardFunction(args...);
```

Comment:
I just came up with this cast fix to unbreak the i386 build of OpenBSD's [security/web-eid-app](https://github.com/openbsd/ports/tree/master/security/web-eid-app) port:
```
In file included from /pobj/web-eid-app-2.3.1/web-eid-app-2.3.1/lib/libelectronic-id/lib/libpcsc-cpp/src/SmartCard.cpp:25:
In file included from /pobj/web-eid-app-2.3.1/web-eid-app-2.3.1/lib/libelectronic-id/lib/libpcsc-cpp/src/Context.hpp:28:
/pobj/web-eid-app-2.3.1/web-eid-app-2.3.1/lib/libelectronic-id/lib/libpcsc-cpp/src/SCardCall.hpp:56:10: error: case value evaluates to -2146435043, which cannot be narrowed to type 'uint32_t' (aka 'unsigned int') [-Wc++11-narrowing]
    case SCARD_E_NO_SERVICE:
         ^
/usr/local/include/PCSC/pcsclite.h:165:29: note: expanded from macro 'SCARD_E_NO_SERVICE'
#define SCARD_E_NO_SERVICE              ((LONG)0x8010001D) /**< The Smart card resource manager is not running. */
                                        ^
```